migrations/versions/da5c6c740257_event_model_rename_has_sessions_.py

```diff
@@ -0,0 +1,27 @@
+"""Event model rename has_sessions_speakers to is_sessions_speakers_enabled
+
+Revision ID: da5c6c740257
+Revises: a6d6f559406f
+Create Date: 2017-06-21 17:31:33.777839
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'da5c6c740257'
+down_revision = 'a6d6f559406f'
```

Comment:
The last head I can see is 76c949b1235f. And in upstream there exists no migration with a6d6f559406f